x/observer/keeper/msg_server_update_crosschain_flags.go

```diff
@@ -0,0 +1,48 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/zeta-chain/zetacore/x/observer/types"
+)
+
+// UpdateCrosschainFlags updates the crosschain related flags.
+// Only the admin policy account is authorized to broadcast this message.
+func (k msgServer) UpdateCrosschainFlags(goCtx context.Context, msg *types.MsgUpdateCrosschainFlags) (*types.MsgUpdateCrosschainFlagsResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	// check permission
+	if msg.Creator != k.GetParams(ctx).GetAdminPolicyAccount(types.Policy_Type_stop_inbound_cctx) {
+		return &types.MsgUpdateCrosschainFlagsResponse{}, types.ErrNotAuthorizedPolicy
+	}
+
+	// check if the value exists
+	flags, isFound := k.GetCrosschainFlags(ctx)
+	if !isFound {
+		flags = *types.DefaultCrosschainFlags()
+	}
+
+	// update values
+	flags.IsInboundEnabled = msg.IsInboundEnabled
+	flags.IsOutboundEnabled = msg.IsOutboundEnabled
```

Comment:
Maybe replace similar behavior for inbound and outbound flags as well, so that the user can just set one if needed?

Reply:
We can do that

Reply:
We can't directly have a bool pointer in proto, we would need to define a wrapper or have another bool like `IsInboundEnabledUpdated` which I think would complexify the API.

Reply:
But we could define a structure
```
struct TxEnabled{
  IsInboundEnabled bool
  IsOutboundEnabled bool
}
```
To give the ability to update the gas price increase params without touching the inbound/outbound enable status